MAIN.ASM

```diff
@@ -55,6 +57,8 @@ endif
     call print_str, offset finished_str2
     call print_newline
 
+    call uninstall_keyboard_handler

```

Comment:
Maybe create a general destruct() function in the INIT.ASM file and add this line in there.